app/src/main/java/com/hedvig/app/feature/home/model/HomeItemsBuilder.kt

```diff
@@ -50,10 +49,9 @@ class HomeItemsBuilder(
                 ).toTypedArray()
             )
         )
-        if (featureManager.isFeatureEnabled(Feature.MOVING_FLOW)) {
```

Comment:
Now we will always show the moving flow button - but take the user to the chat instead of the embark flow.